qiime2/plugin/plugin.py

```diff
@@ -67,9 +69,10 @@ def __init__(self, name, version, website, package, citation_text=None,
 
         self.formats = {}
         self.views = {}
-        self.types = {}
+        self.type_fragments = {}
         self.transformers = {}
         self.type_formats = []
+        self.importable_types = set()
```

Comment:
Please rename:

```suggestion
        self.types = set()
```

:pineapple: